.github/workflows/testing.yml

```diff
@@ -214,6 +214,22 @@ jobs:
           fetch-depth: 0
         if: matrix.tarantool != 'brew' && steps.cache.outputs.cache-hit != 'true'
 
+      - name: Patching tarantool 1.10.x for succesful build
+        run: |
+          # These steps fix the problem with tarantool build described in #6576
+          git diff -p d83e190 ${T_SRCDIR}/src/trivia/util.h > fix.patch
+          patch -p1 -R < fix.patch ${T_SRCDIR}/src/trivia/util.h
+        if: contains(matrix.tarantool, '1.10.') && steps.cache.outputs.cache-hit != 'true'
+
+      - name: Patching tarantool 2.4.x 2.5.x 2.6.x for succesful build
+        run: |
+            # These steps fix the problem with tarantool build described in #6576
+            touch temp
+            echo '#include <stdalign.h>' > temp
+            cat ${T_SRCDIR}/src/trivia/util.h >> temp
+            cat temp > ${T_SRCDIR}/src/trivia/util.h
+        if: matrix.runs-on == 'macos-11.0' && (contains(matrix.tarantool, '2.5.') || contains(matrix.tarantool, '2.4.') || contains(matrix.tarantool, '2.6.')) && steps.cache.outputs.cache-hit != 'true'
```

Comment:
```sh
git show 11e87877df9001a4972019328592d79d55d1bb01 | patch -p1
```

It succeeds on 1.10.7 and 2.4.1 for me. It seems you had some problem with just applying a patch. How can I reproduce it.

Reply:
BTW, I would use the master commit (`11e87877df9001a4972019328592d79d55d1bb01`) instead of the cherry-pick to 1.10 (`d83e1901dc885234fe59eb326748c4d71e106c71`), but that's purely stylistic suggestion.

Reply:
Thank you, this command applies the patch better. These steps are replaced in the latests update.